style.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800&display=swap');
+
+body {
+  margin: 0;
+  padding: 24px;
+  font-family: 'Poppins', sans-serif;
+  display: flex;
+  flex-direction: column;
+  background-color: #f7f7f9;
+  box-sizing: border-box;
+  max-width: 375px;
+}
+
+.bio-section {
+  margin-top: 5%;
+  display: flex;
+  flex-direction: column;
+  background-image: url(img/backg.png);
+  background-size: cover;
+  background-position: center;
+  margin-bottom: 250px;
```

Comment:
- [ ] great job so far you have done a great job on this milestone and we are proud of you ,but you still need some small adjustment to address on this milestone, i would kindly suggest you to stick to tghe design of the original figma template as you can seee your background image not same with the original figma template kindly fix it by adding 
- [ ]    your design
![Screenshot 2023-06-01 171039](https://github.com/josman227/Mobile-Portfolio/assets/92458236/ad2c8a32-0a86-4021-8c8c-e4a8523f8946)

- [ ] given design
![Screenshot 2023-06-01 172333](https://github.com/josman227/Mobile-Portfolio/assets/92458236/3fec1b37-6f72-426a-90f1-456687fe7314)